src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -1167,3 +1256,76 @@ bool emberAfPluginDoorLockGetFingerVeinCredentialLengthConstraints(chip::Endpoin
  * @return false on failure, true on success.
  */
 bool emberAfPluginDoorLockGetFaceCredentialLengthConstraints(chip::EndpointId endpointId, uint8_t & minLen, uint8_t & maxLen);
+
+/**
+ * @brief This callback is called when the Door Lock server needs to find out
+ * the number of Aliro credential issuer keys supported, since there is no attribute
+ * that represents that value.
```

Comment:
Uh... There is _totally_ an attribute that represents that value.  Please copy/paste one of the credential types that has a corresponding attribute, not whatever was copy/pasted here.

Reply:
i incorrectly assumed this gives the number of credential issuer keys currently supported by the lock - going by the API name - GetNumberOfAliroCredentialIssuerKeyCredentialsSupported. Now that you pointed it out I see the out param max credentials. will fix.